src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```

Comment:
@Priya91 how do we skip in runs, yet not actually disable them (so -trait would make them run on demand).